SPECS/ruby/macros.rubygems

```diff
@@ -25,6 +25,7 @@ CONFIGURE_ARGS="--with-cflags='%{optflags}' --with-cxxflags='%{optflags}' $CONFI
 gem install \\\
         -V \\\
         --local \\\
+        --bindir /usr/bin \\\
```

Comment:
Just curious- is there a reason we don't use the bindir macro here?

Reply:
Good catch. Have pushed a change to address it